### PR TITLE
Enhancement/skale 2311 Update to skale.py v3.2

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: test_data/create_environment.sh [hunk 1]
```diff
@@ -27,12 +27,14 @@ export DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 docker run \
```
Comment (dmytrotkk) [inline test_data/create_environment.sh:28]: add `docker pull` here to ensure that you're using the latest version

File: tests/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-skale-py==3.0dev8
```
Comment (dmytrotkk) [inline tests/requirements.txt:1]: Great, but 3.1 is already outdated, sorry for this 😢
Use 3.2 instead (re-check all tests)

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: test_data/create_environment.sh (hunk 1 above)
Comment (alexgex) [reply test_data/create_environment.sh:28]: done

File: tests/requirements.txt (hunk 2 above)
Comment (alexgex) [reply tests/requirements.txt:1]: done

**Review by dmytrotkk — APPROVED** [review]

**Pull request merged**